src/eos-updater-poll-common.c

```diff
@@ -313,6 +313,8 @@ get_repo_pull_options (const gchar *url_override,
                          g_variant_new_variant (g_variant_new_int32 (OSTREE_REPO_PULL_FLAGS_COMMIT_ONLY)));
   g_variant_builder_add (&builder, "{s@v}", "refs",
                          g_variant_new_variant (g_variant_new_strv (&ref, 1)));
+  g_variant_builder_add (&builder, "{s@v}", "gpg-verify-summary",
+                         g_variant_new_variant (g_variant_new_boolean (TRUE)));
```

Comment:
Why isn't this doing the same "respect if the config has `gpg-verify-summary=false`" dance as the other commit?